internal/manifests/osbuild.go

```diff
@@ -184,9 +190,7 @@ func (o *OSBuildCreator) applyTemplate(ctx context.Context, osBuildConfig *osbui
 		if err != nil {
 			return nil, nil, err
 		}
-		if kickstartConfigMap != nil {
```

Comment:
Why did you remove this case? The configMap is still being created

Reply:
because kicktsart now is part of edgeinstallerdetals and it cannot be set automatically -this section was moved to the create function as part of `EdgeInstallerImageType`

Reply:
If that's so, then we need to move the creation of the ConfigMap as well.

Reply:
I moved the kickstartConfigMap creation to the EdgeInstallerImageType case, but keep the setting owner in the full flow (I mean [here:](https://github.com/project-flotta/osbuild-operator/pull/143/files#diff-eb685724eddffc42344c24fdd392f723af5051deda96dc5cb59dbfc4f69ff9c2L119) ) because only if the osBuild was created I would like to set the owner.

Reply:
Done - Please resolve if it's fine by you

Reply:
Not sure if it works or not. I guess we'll need to implement the edge-installer and the template to find out. Resolving for now